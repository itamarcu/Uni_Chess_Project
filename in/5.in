game_mode 2
start
move <2,D> to <3,E>
move <2,D> to <3,D>
move <7,B> to <6,B>
move <1,C> to <2,C>
move <8,C> to <6,A>
move <2,D> to <4,A>
move <2,D> to <5,A>
move <6,A> to <5,A>
move <1,A> to <7,A>
move <1,E> to <2,D>
move <8,F> to <6,H>
move <8,F> to <6,H>
quit
