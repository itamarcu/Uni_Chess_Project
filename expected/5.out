 Chess
-------
Specify game settings or type 'start' to begin a game with the current settings:
Game mode is set to 2-player
Starting game...
8| R N B Q K B N R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m m m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
Illegal move
Enter your move (white player):
8| R N B Q K B N R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ m _ _ _ _ |
2| m m m _ m m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R N B Q K B N R |
7| M _ M M M M M M |
6| _ M _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ m _ _ _ _ |
2| m m m _ m m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
Illegal move
Enter your move (white player):
The specified position does not contain your piece
Enter your move (white player):
The specified position does not contain your piece
Enter your move (white player):
The specified position does not contain your piece
Enter your move (white player):
The specified position does not contain your piece
Enter your move (white player):
Illegal move
Enter your move (white player):
8| R N B Q K B N R |
7| M _ M M M M M M |
6| _ M _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ m _ _ _ _ |
2| m m m k m m m m |
1| r n b q _ b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
Illegal move
Enter your move (black player):
Illegal move
Enter your move (black player):
Exiting...
